graph/telemetry/istio/istio.go

```diff
@@ -656,6 +657,7 @@ func promQuery(query string, queryTime time.Time, api prom_v1.API) model.Vector
 	log.Tracef("Graph query:\n%s@time=%v (now=%v, %v)\n", query, queryTime.Format(graph.TF), time.Now().Format(graph.TF), queryTime.Unix())
 
 	promtimer := internalmetrics.GetPrometheusProcessingTimePrometheusTimer("Graph-Generation")
+	log.Infof("GraphGen: %s", query)
```

Comment:
you sure you want this at the "Info" level? This seems noisy.

Reply:
I don't want it at all :)  Thanks.